how-to-guides/third-party/README.md

```diff
@@ -0,0 +1 @@
+# Third Party
```

Comment:
can you add a table here like these ones? https://github.com/devzero-inc/product-docs/blob/staging/how-to-guides/cloud-services/README.md?plain=1